packages/x-codemod/src/v6.0.0/pickers/replace-tabs-props/expected.spec.js

```diff
@@ -0,0 +1,33 @@
+<DateTimePicker
+  componentsProps={{
+    tabs: {
+      dateIcon: <LightModeIcon />,
+    },
+  }}
+/>;
+
+<DateTimePicker
+  componentsProps={{
+    tabs: {
+      hidden: false,
+      dateIcon: <LightModeIcon />,
+      timeIcon: <AcUnitIcon />,
+    },
+  }} />;
+
+<DateTimePicker
+  componentsProps={{
+    actionBar: {
+      actions: [],
+    },
+
```

Comment:
I would encourage users to run prettier after the codemod O:)

If you have an idea about how to solve this problem, why not. Otherwise, I prefer not to fix it since writing codemod is time-consuming (I fear half a day of Googling to find how to remove those empty lines)

Reply:
Indeed, I experimented a bit with `lineTerminator` in [recast options](https://github.com/benjamn/recast/blob/master/lib/options.ts), it seems to control it, but there does not seem to be a smart option to control it's value conditionally, nor does it seem to support custom prettier config, where we could specify that empty lines should be removed. 🤔 
TL.DR.: The current solution seems sufficient 👍